### PR TITLE
Address test plan feedback reported by community group

### PR DESCRIPTION
**Pull request by IsaDC**
[Preview Tests](https://raw.githack.com/w3c/aria-at/cg-feecback/build/index.html)

#### Select-Only Combobox
1. Tests 1-6.  Remove previous instructions and replace them with the new format. This makes clear to the tester where to go and where the cursor should be. Addresses issue #478.
2. Tests 1-6. Add the exact number of commands required to perform an action, such as navigating forwards to a combobox.  Addresses internal feedback.
3. Tests 10-14, 19-20, 22, 24, 27 and 29.  Change assertion of selected option to not conveyed.  Addresses internal feedback.

#### Editable Combobox with Both List and Inline Autocomplete
1. Tests 1-6, 10-15.  Remove previous instructions and replace them with the new format. This makes clear to the tester where to go and where the cursor should be. Addresses internal feedback.
2. Tests 1-6, 10-15.  Add the exact number of commands required to perform an action, such as navigating forwards to a combobox.  Addresses internal feedback.

#### Minimal Data Grid

1. Test 1. Add the exact commands required to perform an action, such as navigate forwards to a grid.
2. Tests 18-35. Fix typo (column)
3. Tests 36 to 47. Remove assertion: Text of the column header ('Date') is conveyed. Addresses issue #538
4.  Tests 30-35.  Fix typo (cell)
5.  Test 35.  Remove SHIFT_CTRL_OPT_CMD_L command

### CODE REVIEW

File: tests/combobox-autocomplete-both-updated/data/tests.csv [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+13,"Navigate backwards to a filled in, collapsed combobox in interaction mode","JAWS,NVDA",interaction,navigate backwards to filled in collapsed combobox,moveFocusAfterComboboxAndSetValueToAlabama,"sets focus on a link after the Combobox, and sets the combobox value to 'Alabama'",combobox aria-expanded aria-autocomplete,With focus on the 'Navigate backwards from here' link, navigate to the 'State' Combobox.,Role 'combobox' is conveyed,Name of the combobox ('State') is conveyed,State of the combobox (collapsed) is conveyed,Text of the selected option ('Alabama') is conveyed,The ability to enter or edit text is conveyed,State of the text in the input (selected) is conveyed,The presence of autocomplete behavior is conveyed
```
Comment (howard-e) [inline tests/combobox-autocomplete-both-updated/data/tests.csv:14]: `With focus on the 'Navigate backwards from here' link, navigate to the 'State' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-autocomplete-both-updated/data/tests.csv [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+14,"Navigate forwards to a filled in, collapsed combobox",voiceover_macos,interaction,navigate forwards to filled in collapsed combobox,moveFocusBeforeComboboxAndSetValueToAlabama,"sets focus on a link before the Combobox, and sets the combobox value to 'Alabama'",combobox aria-expanded aria-autocomplete,With focus on the 'Navigate forwards from here' link, navigate to the 'State' Combobox.,Role 'combobox' is conveyed,Name of the combobox ('State') is conveyed,State of the combobox (collapsed) is conveyed,Text of the selected option ('Alabama') is conveyed,The ability to enter or edit text is conveyed,State of the text in the input (selected) is conveyed,The presence of autocomplete behavior is conveyed
```
Comment (howard-e) [inline tests/combobox-autocomplete-both-updated/data/tests.csv:15]: `With focus on the 'Navigate forwards from here' link, navigate to the 'State' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-autocomplete-both-updated/data/tests.csv [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+15,"Navigate backwards to a filled in, collapsed combobox",voiceover_macos,interaction,navigate backwards to filled in collapsed combobox,moveFocusAfterComboboxAndSetValueToAlabama,"sets focus on a link after the Combobox, and sets the combobox value to 'Alabama'",combobox aria-expanded aria-autocomplete,With focus on the 'Navigate backwards from here' link, navigate to the 'State' Combobox.,Role 'combobox' is conveyed,Name of the combobox ('State') is conveyed,State of the combobox (collapsed) is conveyed,Text of the selected option ('Alabama') is conveyed,The ability to enter or edit text is conveyed,State of the text in the input (selected) is conveyed,The presence of autocomplete behavior is conveyed
```
Comment (howard-e) [inline tests/combobox-autocomplete-both-updated/data/tests.csv:16]: `With focus on the 'Navigate backwards from here' link, navigate to the 'State' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-select-only/data/commands.csv [hunk 4]
```diff
@@ -1,14 +1,14 @@
 testId,task,mode,at,commandA,commandB,commandC,commandD,commandE,commandF
-1,navigate forwards to collapsed select-only combobox,reading,JAWS,F,C,TAB,DOWN,,
+1,navigate forwards to collapsed select-only combobox,reading,JAWS,F,C	,TAB,"DOWN,DOWN,DOWN",,
```
Comment (howard-e) [inline tests/combobox-select-only/data/commands.csv:2]: I believe _space_ after `C` should be removed.

File: tests/combobox-select-only/data/tests.csv [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
+1,Navigate forwards to a collapsed select-only combobox in reading mode,"JAWS,NVDA",reading,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With the reading cursor on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```
Comment (howard-e) [inline tests/combobox-select-only/data/tests.csv:2]: `With the reading cursor on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-select-only/data/tests.csv [hunk 6]
```diff
@@ -1,33 +1,33 @@
...
+2,Navigate backwards to a collapsed select-only combobox in reading mode,"JAWS,NVDA",reading,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With the reading cursor on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```
Comment (howard-e) [inline tests/combobox-select-only/data/tests.csv:3]: `With the reading cursor on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-select-only/data/tests.csv [hunk 7]
```diff
@@ -1,33 +1,33 @@
...
+3,Navigate forwards to a collapsed select-only combobox in interaction mode,"JAWS,NVDA",interaction,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+4,Navigate backwards to a collapsed select-only combobox in interaction mode,"JAWS,NVDA",interaction,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```
Comment (howard-e) [inline tests/combobox-select-only/data/tests.csv:5]: `With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-select-only/data/tests.csv [hunk 8]
```diff
@@ -1,33 +1,33 @@
...
+5,Navigate forwards to a collapsed select-only combobox,voiceover_macos,interaction,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```
Comment (howard-e) [inline tests/combobox-select-only/data/tests.csv:6]: `With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.

File: tests/combobox-select-only/data/tests.csv [hunk 9]
```diff
@@ -1,33 +1,33 @@
...
+6,Navigate backwards to a collapsed select-only combobox,voiceover_macos,interaction,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```
Comment (howard-e) [inline tests/combobox-select-only/data/tests.csv:7]: `With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.

**Review by howard-e — CHANGES_REQUESTED** [review]

File: tests/combobox-select-only/data/tests.csv (hunk 9 above)
Comment (IsaDC) [reply tests/combobox-select-only/data/tests.csv:7]: Thanks.  Working on it.

**Comment by jscholes** [comment]
@howard-e The build process is still failing here, which is fine.  But the error I'm seeing is:

`ERROR: Unhandled exception thrown while processing "combobox-autocomplete-both-updated".`

It then instructs me to, `Run 'npm run create-all-tests -- --verbose' to view detailed information on error.`.  That only gives me the same sort of vague, useless stack traces we were getting before, without indicating where or what the error actually is.  So, not sure how to proceed.

Note that I'm not asking you where the error is (although that would be useful to know).  I'm asking why, despite several PRs being merged, the error info is now no more helpful than it was before?  If anything, it's less helpful, because it isn't included in the GitHub Actions output.

**Comment by jesdaigle** [comment]
@jscholes [issue 615](https://github.com/w3c/aria-at/pull/615) solved the issue that it was written to solve. There have been no other issues filed or requests made to do anything differently and what you're eluding to is a very specific case which would need to be handled separately (the stack trace you reference would helpful to be able identify the actual problem). To solve the specific issue you're having you can follow Howard's advice and use double quotes due to the mid-sentence comma in the instructions.

**Comment by jscholes** [comment]
Hi @jesdaigle, unfortunately @IsaDC is currently out of the office due to illness.  However, following my most recent commit, I think all of the feedback should be addressed.  Thank you to @howard-e for working through and providing the input!

The test build process seems to have completed successfully, so hopefully this should be good to go.  The preview link is giving me a 404, which may be related to other work happening elsewhere to move to Netlify.  But, we can address any outstanding editorial items later, if they still exist.

Let me know if anything else is needed on this one.

**Comment by jesdaigle** [comment]
Thank you @jscholes. I hope @IsaDC is better soon.

**Comment by jesdaigle** [comment]
@jscholes The 404 should go away after a hard refresh.

**Comment by jscholes** [comment]
@jesdaigle Yep, you're right.  Things look good to me from a quick scan.

**Review by howard-e — APPROVED** [review]

**Pull request merged**